crane/data.py

```diff
@@ -106,6 +115,9 @@ def load_all(app):
     :param app: the flask application
     :type  app: flask.Flask
     """
+    global response_data_v2
+    repos_v2 = {}
+
     global response_data
     repos = {}
```

Comment:
I'd suggest renaming repos to v1_repos and repos_v2 to v2_repos for consistency.